CMakeLists.txt

```diff
@@ -444,6 +444,15 @@ if(__BUILD_FOR_R)
   endif(MSVC)
 endif(__BUILD_FOR_R)
 
+# fmtlib/fmt
+#if (NOT BUILD_STATIC_LIB)
+#  OPTION(BUILD_SHARED_LIBS "Build shared libs" ON)  # fmt reads this flag.
+#endif(NOT BUILD_STATIC_LIB)
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -funroll-loops")
```

Comment:
I'm not CMake expert at all, but I believe it could be done by something like following:
```
set_property(TARGET fmt::fmt PROPERTY POSITION_INDEPENDENT_CODE)
```
Suggestion is based on https://cliutils.gitlab.io/modern-cmake/chapters/packages/OpenMP.html.

Reply:
Ok I will try replacing it monday ;)

Reply:
Fixed :) Had to change some stuff due to ALIAS targets.